docs/_documentations/troubleshooting.md

```diff
@@ -416,7 +416,48 @@ These steps reproduce the issue:
 6. Click **Finish**. The OpenAPI generator fails if the folder doesn't already exist.
 
 **Workaround:**
+For the VS Code extension: 
 - Manually create the output folder before you start the OpenAPI generator wizard. In the wizard, you can manually edit the **Output folder** text field. Ensure that the path points to a valid folder in the project.
 - Or you can use the **Browse...** button to select the folder that you want to use. The **Output folder** field is updated with the path.
```

Comment:
Correction to my original request. Remove this second point completely:
"Or you can use the **Browse...** button to select the folder that you want to use. The **Output folder** field is updated with the path."

Reply:
Removed second bullet, reworked lead in sentence and first bullet to form a paragraph.